src/build_loss_objective.jl

```diff
@@ -8,7 +8,7 @@ end
 (f::DiffEqObjective)(x) = f.cost_function(x)
 (f::DiffEqObjective)(x,y) = f.cost_function2(x,y)
 
-function build_loss_objective(prob::DEProblem,alg,loss,regularization;mpg_autodiff = false,
+function build_loss_objective(prob::DEProblem,alg,loss;regularization=nothing,mpg_autodiff = false,
```

Comment:
default argument, not a keyword argument. `build_loss_objective(prob::DEProblem,alg,loss,regularization=nothing;`, see the difference with the `;`. Keyword arguments won't dispatch so that'll be somewhat of an issue (though actually not that big of a deal). But this will feel more natural, i.e. you usually leave it off but add it when you want.